### PR TITLE
added missing rake task alias

### PR DESCRIPTION
**Pull request by trkrameshkumar**

### CODE REVIEW

File: plugins/rails/rails.plugin.zsh [hunk 1]
```diff
@@ -62,6 +62,17 @@ alias rrg='rake routes | grep'
 alias rt='rake test'
 alias rmd='rake middleware'
 alias rsts='rake stats'
+alias rtasks = 'rake --tasks'
```
Comment (sachin21) [inline plugins/rails/rails.plugin.zsh]: It does not works.
Could you change to this?
before
```
alias rtc = 'rake tmp:clear'
```

after
```
alias rtc='rake tmp:clear'
```

**Comment by trkrameshkumar** [comment]
@sachin21 syntax fixed, pls check

File: plugins/rails/rails.plugin.zsh [hunk 2]
```diff
@@ -62,6 +62,17 @@ alias rrg='rake routes | grep'
...
 alias rsts='rake stats'
+alias rtasks='rake --tasks'
+alias ra='rake about'
+alias rdv='rake db:version'
+alias rtcc='rake tmp:cache:clear'
+alias rtssc='rake tmp:sessions:clear'
+alias rtsc='rake tmp:sockets:clear'
+alias rtc='rake tmp:clear'
+alias rtmp='rake tmp:create'
+alias rst='rake secret'
```
Comment (sachin21) [inline plugins/rails/rails.plugin.zsh]: `rst` alias is duplicated with https://github.com/robbyrussell/oh-my-zsh/blob/master/plugins/repo/repo.plugin.zsh#L17

**Comment by trkrameshkumar** [comment]
@sachin21 closing this PR and giving new PR  with changes

**Pull request closed without merging**